src/celestia/celestiacore.cpp

```diff
@@ -3966,6 +3966,32 @@ bool CelestiaCore::initSimulation(const fs::path& configFileName,
     }
 
     sim = new Simulation(universe);
+
+    if (config->defaultGoToTime >= 0.0)
+    {
+        sim->setDefaultGoToTime(config->defaultGoToTime);
+    }
+
+    if (config->defaultCenterTime >= 0.0)
+    {
+        sim->setDefaultCenterTime(config->defaultCenterTime);
+    }
+
+    if (config->defaultInterpolationStart >= 0.0 && config->defaultInterpolationEnd <= 1.0 && config->defaultInterpolationEnd >= config->defaultInterpolationStart)
+    {
+        sim->setDefaultInterpolation(config->defaultInterpolationStart, config->defaultInterpolationEnd);
+    }
+
+    if (config->altInterpolationStart >= 0.0 && config->altInterpolationEnd <= 1.0 && config->altInterpolationEnd >= config->altInterpolationStart)
+    {
+        sim->setAlternativeInterpolation(config->altInterpolationStart, config->altInterpolationEnd);
+    }
+
+    if (config->defaultAccelTime >= 0.1 && config->defaultAccelTime <= 1.0)
```

Comment:
i think it's good to provide constants like k_defaultAccelTimeMin = 0.1 & k_defaultAccelTimeMax = 1.0 instead of hardcoding 0.1 & 1.0 here and there